### PR TITLE
Some progress for front page layout

### LINKED ISSUE #65: Home page
Working on it!

### PR DESCRIPTION
**Pull request by epicfaace**
Fixes #65.

Show front page layout like:

image

### CODE REVIEW

**Comment by hesyifei** [comment]
> @hesyifei ok, initially I didn't know you were already editing the sass and assumed that we would edit css temporarily, then use a plugin such as [wp-scss](https://wordpress.org/plugins/wp-scss/) to compile it server-side.
> 
> What command-line tool/exact command are you using right now for compiling the sass? So we can standardize it and add it to the readme.

We are using Grunt now! 🥳

https://github.com/TheStanfordDaily/stanforddaily-website/tree/custom-theme#theme-development

**Comment by epicfaace** [comment]
@hesyifei more progress 😁 

image

**Comment by epicfaace** [comment]
Latest:

image

File: wp-content/themes/thestanforddaily/functions.php [hunk 1]
```diff
@@ -123,15 +123,17 @@ function tsd_scripts() {
...
 	wp_enqueue_script( 'jquery' );
-
 	wp_enqueue_script( 'tsd-navigation', get_template_directory_uri() . '/js/navigation.js', array(), '20151215', true );
-
 	wp_enqueue_script( 'tsd-skip-link-focus-fix', get_template_directory_uri() . '/js/skip-link-focus-fix.js', array(), '20151215', true );
 
 	if ( is_singular() && comments_open() && get_option( 'thread_comments' ) ) {
-		wp_enqueue_script( 'comment-reply' );
+		// We are using Disqus
+		//wp_enqueue_script( 'comment-reply' );
```
Comment (epicfaace) [inline wp-content/themes/thestanforddaily/functions.php]: @hesyifei why?

File: wp-content/themes/thestanforddaily/template-parts/content.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
 		if ( 'post' === get_post_type() ) :
 			?>
 			<div class="entry-meta">
-				<?php tsd_posted_by(); ?> &mdash; <?php tsd_posted_on(); //tsd_comments_count(); ?>
+				<?php tsd_posted_by(); ?> &mdash; <?php tsd_posted_on(); tsd_comments_count(); ?>
```
Comment (epicfaace) [inline wp-content/themes/thestanforddaily/template-parts/content.php]: Cool. Eventually we will need to update tsd_comments_count() to actually pull from disqus though. Right now it's kind of useless.

Comment (hesyifei) [reply wp-content/themes/thestanforddaily/template-parts/content.php]: Disqus plugin take care of that and it's working correctly right now.

File: wp-content/themes/thestanforddaily/functions.php (hunk 1 above)
Comment (hesyifei) [reply wp-content/themes/thestanforddaily/functions.php]: This script is used for native WP comments. We are using Disqus, so no need to load this.

**Pull request merged**